lua/acf/base/acf_globals.lua

```diff
@@ -128,6 +128,18 @@ do -- ACF global vars
 	ACF.RefillDistance     = 300 --Distance in which ammo crate starts refilling.
 	ACF.RefillSpeed        = 700 -- (ACF.RefillSpeed / RoundMass) / Distance
 	ACF.RefuelSpeed        = 20 -- Liters per second * ACF.FuelRate
+
+	-- Mobility
+	-- Use this Starfall chip to help design torque curves https://gist.github.com/CheezusChrust/7ccce5f5196d3adc95ab9573009f735a
```

Comment:
Could maybe be included as a dupe?

Reply:
In the future I was thinking it'd be nice to have a "View Details" button in the engine menu that'd bring up a dyno chart drawn to a rendertarget, would be a better alternative to people having to manually find the torque curve values each engine uses in the files. An included dupe could work in the mean time though.